src/features/todos.ts

```diff
@@ -4,5 +4,10 @@ import { Todo } from '../types/Todo';
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    add: (todos, action) => [...todos, ...action.payload],
```

Comment:
The `add` reducer appends new todos to the existing list. Ensure that this reducer is correctly dispatched with the appropriate payload (an array of `Todo` objects) to update the state as expected.